src/features/content-explorer/content-explorer/ContentExplorer.scss

```diff
@@ -72,22 +72,50 @@
         }
     }
 
+    .bdl-ContentExplorer-subheader {
+        display: flex;
+        align-items: center;
+
+        .checkbox-container,
+        .toggle-container {
+            margin: 0;
+        }
+
+        .content-explorer-select-all-checkbox {
+            margin-right: 4.45 * $bdl-grid-unit;
```

Comment:
how was this calculated? could we do 18px?

Reply:
We could do 18 px as well. I just thought it was recommended to use bdl-grid-units since the previous code for ContentExplorerSelectAll uses it